x-pack/plugins/threat_intelligence/server/utils/indicator_name.ts

```diff
@@ -40,6 +40,7 @@ const mappingsArray: Mappings = [
       RawIndicatorFieldId.FileImphash,
       RawIndicatorFieldId.FilePehash,
       RawIndicatorFieldId.FileVhash,
+      RawIndicatorFieldId.FileTelfhash,
```

Comment:
this was missing from https://github.com/elastic/kibana/blob/main/x-pack/plugins/threat_intelligence/common/types/indicator.ts#L63